### PR TITLE
Feat:Language change in pages

### PR DESCRIPTION
**Pull request by rafael-ramirez-orte**
# Description
Language change in pages 

Please include a short summary of the change and which issue is fixed.

### Screenshots, Videos, Gifs.
WhatsApp Image 2023-02-08 at 11 20 00 AM

Add some screenshots, videos, gifs, etc, that may help the reviewer to get a better context of the requested changes.

# Checklist:

- [ ] Pull request owner assigned
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] I have pulled / merged main/develop into this branch

[IL-ID]

### CODE REVIEW

File: page/RedEvat.jsx [hunk 1]
```diff
@@ -182,7 +182,7 @@ const RedEvat = () => {
                     render={({ field: { onChange, onBlur, value } }) => (
                         <TextInput
                             style={styles.inputRed}
-                            placeholder="Fecha Primer EVAT rojo"
+                            placeholder="Date First red EVAT"
```
Comment (JavierRuvalcaba) [inline page/RedEvat.jsx:185]:
```suggestion
                            placeholder="Fisrt red EVAT date"
```

File: page/RedEvat.jsx [hunk 2]
```diff
@@ -352,7 +352,7 @@ const RedEvat = () => {
                     render={({ field: { onChange, onBlur, value } }) => (
                         <TextInput
                             style={styles.inputRed}
-                            placeholder="Hora de intervención"
+                            placeholder="Intervention time"
```
Comment (JavierRuvalcaba) [inline page/RedEvat.jsx:355]:
```suggestion
                            placeholder="Intervention hour"
```

File: page/RedEvat.jsx [hunk 3]
```diff
@@ -386,7 +386,7 @@ const RedEvat = () => {
                     render={({ field: { onChange, onBlur, value } }) => (
                         <TextInput
                             style={styles.inputRed}
-                            placeholder="Duración de EVAT rojo(Horas)"
+                            placeholder="Red EVAT duration"
```
Comment (JavierRuvalcaba) [inline page/RedEvat.jsx:389]:
```suggestion
                            placeholder="Red EVAT duration (hours)"
```

File: page/RedEvat.jsx [hunk 4]
```diff
@@ -454,7 +454,7 @@ const RedEvat = () => {
                     render={({ field: { onChange, onBlur, value } }) => (
                         <TextInput
                             style={styles.inputRed}
-                            placeholder="Hora de Evento"
+                            placeholder="Event time"
```
Comment (JavierRuvalcaba) [inline page/RedEvat.jsx:457]:
```suggestion
                            placeholder="Event hour"
```

File: page/RedEvat.jsx [hunk 5]
```diff
@@ -505,7 +505,7 @@ const RedEvat = () => {
                     render={({ field: { onChange, onBlur, value } }) => (
                         <TextInput
                             style={styles.inputRed}
-                            placeholder="Hora de Traslado"
+                            placeholder="Transfer time"
```
Comment (JavierRuvalcaba) [inline page/RedEvat.jsx:508]:
```suggestion
                            placeholder="Transfer hour"
```

File: page/DailyForm.jsx [hunk 6]
```diff
@@ -129,7 +129,7 @@ const DailyForm = () => {
 						<TextInput
 							keyboardType="numeric"
 							maxLength={3}
-							placeholder="Frecuencia cardiaca"
+							placeholder="Cardiac frequency"
```
Comment (JavierRuvalcaba) [inline page/DailyForm.jsx:132]:
```suggestion
							placeholder="Heart rate"
```

File: page/DailyForm.jsx [hunk 7]
```diff
@@ -147,7 +147,7 @@ const DailyForm = () => {
 						<TextInput
 							keyboardType="numeric"
 							maxLength={3}
-							placeholder="Frecuencia respiratoria"
+							placeholder="Respiratory rate"
```
Comment (JavierRuvalcaba) [inline page/DailyForm.jsx:150]:
```suggestion
							placeholder="Breathing rate"
```

File: page/LoginPage.jsx [hunk 8]
```diff
@@ -65,16 +56,16 @@ const LoginPage = ({ navigation }) => {
...
                     )}
                     name="password"
                 />
                 <TouchableOpacity
                     style={styles.buttonContainer}
-                    onPress={handleSubmit(onSubmit)}
+                    onPress={handleSubmit((data)=> signIn(data.email, data.password))}
                 >
-                    <Text style={styles.ButtonText}>Ingresar</Text>
+                    <Text style={styles.ButtonText}>Enter</Text>
```
Comment (JavierRuvalcaba) [inline page/LoginPage.jsx:68]:
```suggestion
                    <Text style={styles.ButtonText}>Sign in</Text>
```

File: package-lock.json [hunk 9]
```diff
@@ -6480,6 +6480,38 @@
...
+      "version": "15.7.5",
+      "resolved": "https://registry.npmjs.org/@types/prop-types/-/prop-types-15.7.5.tgz",
+      "integrity": "sha512-JCB8C6SnDoQf0cNycqd/35A7MjcnK+ZTqE7judS6o7utxUCg6imJg3QK2qzHKszlTjcj2cn+NwMB2i96ubpj7w==",
+      "peer": true
+    },
+    "node_modules/@types/react": {
+      "version": "18.0.27",
+      "resolved": "https://registry.npmjs.org/@types/react/-/react-18.0.27.tgz",
+      "integrity": "sha512-3vtRKHgVxu3Jp9t718R9BuzoD4NcQ8YJ5XRzsSKxNDiDonD2MXIT1TmSkenxuCycZJoQT5d2vE8LwWJxBC1gmA==",
+      "peer": true,
```
Comment (JavierRuvalcaba) [inline package-lock.json:6493]: Why are you sending this file?
I think is not necessary for this PR

**Review by JavierRuvalcaba — APPROVED** [review]

**Review by pedro123-67we — APPROVED** [review]

**Review by pedro123-67we — APPROVED** [review]

**Pull request merged**